data/devices/huawei.yml

```diff
@@ -158,4 +158,41 @@
       - /dev/block/mmcblk0p28
     recovery:
       - /dev/block/platform/hi_mci.0/by-name/recovery
-      - /dev/block/mmcblk0p29
\ No newline at end of file
+      - /dev/block/mmcblk0p29
+      
+      
+      name: Huawei P9 Lite
```

Comment:
I'm really confused, what should I edit, the p9 lite had same structure as the huawei honor 6, only codename,Id are different...you're asking me to modify and I'm lost, do you want logs?